go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/jtblin/kube2iam
 
-go 1.15
+go 1.22.0
```

Comment:
Why locking to `1.22.0`? It looks like the latest `1.22` was `1.22.12`.

And is the reason to stick to 1.22 instead of 1.23 or 1.24 for stability?

Reply:
i believe this specifies the minimum version (compared to `toolchain`), which matches what client-go uses in their release-1.30 go.mod file